app/javascript/packs/logged-in-user.js

```diff
@@ -40,3 +42,8 @@ document.addEventListener("turbolinks:before-cache", function() {
   const svelteComponents = document.querySelectorAll("[data-svelte-component]")
   svelteComponents.forEach(element => element.innerHTML = null)
 })
+
+document.addEventListener("turbolinks:click", () => {
```

Comment:
I'm a bit confused why this is here, isn't this the same as the previous line, but on `click` instead of `before-cache`?

Reply:
For some reason I found that Turbolinks was still showing me a cached version of the page with Svelte components and this addition happened to resolve the issue. Unsure if this applies to a Heroku deployment. I may deploy to my own staging environment to have a look.